deploy/cr.yaml

```diff
@@ -158,6 +158,8 @@ spec:
 #        - 10.0.0.0/8
 #      serviceAnnotations:
 #        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+#      serviceLabels:
```

Comment:
I recently created this spec for aligning exposure: https://docs.google.com/document/d/112X1B_Z1yplhegUb8SyJ9y4G2RQPFfBy9dCUlkLn2dc/edit#

Can we call it:
```
expose:
  labels:
```
right now?

Reply:
@spron-in if it is not urgent I prefer to do it in next release under K8SPSMDB-739 task